tests/Core/Tokenizers/Tokenizer/RecurseScopeMapDefaultKeywordConditionsTest.inc

```diff
@@ -201,3 +201,26 @@ class Foo {
     /* testMethodDeclaration */
     public function default() {}
 }
+
+function switchAndDefaultSharingScopeCloser($i) {
+    switch ($i):
+        /* testSwitchAndDefaultSharingScopeCloser */
+        default:
+            echo 'one';
+    endswitch;
+}
+
+function switchDefaultNestedIfWithAndWithoutBraces($i, $foo, $baz) {
+    switch ($i) {
+        /* testSwitchDefaultNestedIfWithAndWithoutBraces */
```

Comment:
The test has braces for all if/else leaves, while the point of the test was that it would not (also see the marker name)....

```php
        if ($foo) {
            return true;
        } elseif ($baz)
            return true;
        else {
            echo 'else';
        }
```

Reply:
Done in fb77fa33970ba0b9e5367c7bc9db9a0031c94b86